### PR TITLE
Update outdated information in docs 

### PR DESCRIPTION
**Pull request by jovyntls**
**What is the purpose of this pull request?**

- [x] Documentation update
- [ ] Bug fix
- [ ] Feature addition or enhancement
- [ ] Code maintenance
- [ ] Others, please explain:

**Overview of changes:**
Mainly minor grammatical fixes, along with some more notable fixes:
* Removed the `effect` attribute from these examples in the docs: 
	Screen Shot 2022-02-16 at 4 28 23 PM
	* `effect` is currently undocumented, and there seems to be no implementation for it either
	* `effect=fade` and `effect=scale` seem to not do anything; related discussion: #611
* The `effect` attribute was included in a few test files as well, which are removed in #1783 
* Use the codeAndOutput boilerplate in `docs/userGuide/usingHtmlJavaScriptCss.md`, to follow how the rest of the docs displays code + output

**Anything you'd like to highlight / discuss:**
~~Not sure if removing the `effect` attribute from the tests is out of scope of this PR; if it is I'll revert my last commit!~~ Addressed in #1783

**Testing instructions:**
N/A

**Proposed commit message: (wrap lines at 72 characters)**

```
Update outdated information in docs

Currently, the effect attribute is not implemented in popovers and 
tooltips, which means that using this attribute has no effect. 
However, the effect attribute is currently included in the examples 
for popovers. This may be confusing for readers. 

Let's remove all occurrences of the effect attribute from the docs 
in places where it has no effect. This will prevent readers from 
mistaking it as a valid attribute. 

Other minor grammatical errors were fixed as well. 
```

---

**Checklist:** :ballot_box_with_check:

- [ ] Updated the documentation for feature additions and enhancements
- [ ] Added tests for bug fixes or features
- [ ] Linked all related issues
- [x] No unrelated changes

### CODE REVIEW

**Review by tlylt — COMMENTED** [review]
Looks good to me.

**Review by jonahtanjz — APPROVED** [review]
LGTM 👍 

> Not sure if removing the effect attribute from the tests is out of scope of this PR; if it is I'll revert my last commit!

Yup it's better to split up this PR to not confuse anyone looking through the commit history :) You may also want to update the commit message to include why `effect` was removed from the doc.

**Comment by jovyntls** [comment]
@jonahtanjz I've removed the changes to the tests and edited the commit messgae. Thanks for the suggestion!

**Comment by jonahtanjz** [comment]
Thanks @jovyntls!

Everything looks good. Will merge this once the conflicts has been resolved :)

**Review by ryoarmanda — APPROVED** [review]
LGTM!

**Comment by tlylt** [comment]
@all-contributors please add @jovyntls for code, doc

File: packages/cli/test/functional/test_site_algolia_plugin/index.md [hunk 1]
```diff
@@ -30,12 +30,12 @@
 
 **Popover content should have algolia-no-index class**
 
-<popover effect="fade" header="Title" placement="top">
+<popover header="Title" placement="top">
```
Comment (jonahtanjz) [inline packages/cli/test/functional/test_site_algolia_plugin/index.md:33]: I believe you accidentally included the changes for the test and template files in this PR 😅

Comment (jovyntls) [reply packages/cli/test/functional/test_site_algolia_plugin/index.md]: Sorry I missed it previously, I've made the changes!

**Pull request merged**